Add new connection sequence diagram

dznz opened the pull request:
And replace example DID's with [HIPE-0014 notation][1].

[1]: https://github.com/hyperledger/indy-hipe/tree/master/text/0014-ssi-notation

Commit:
Add new connection sequence diagram

And replace example DID's with [HIPE-0014 notation][1].

[1]: https://github.com/hyperledger/indy-hipe/tree/master/text/0014-ssi-notation

tplooker commented on text/agent-message-routing/README.md:
```diff
@@ -22,12 +22,14 @@ Routing records will underpin the ability to successfully deliver an agent messa
 
 Bob and Alice want to connect so they can exchange messages.
 
+![New Connection Sequence Diagram](new-connection-sequence.png)
```
Wondering if seen as the example below explains the process as it goes, whether this diagram is best stated after the text description?

tplooker commented on text/agent-message-routing/README.md:
```diff
@@ -42,54 +44,54 @@ Note - lets also assume that Alice's agent (for Bob) is directly contactable and
+Verkey = `B.vk@B:A`
 
-**Routing Record Setup**
+**Routing Record Setup** - Steps 2-4
```
If we did move the diagram then maybe put in brackets here on the end of this line `(Note the steps refer to those noted on the UML diagram below)`

tplooker commented on text/agent-message-routing/README.md:
````diff
@@ -42,54 +44,54 @@ Note - lets also assume that Alice's agent (for Bob) is directly contactable and
 ```json
 {
  "@type": "did:sov:BzCbsNYhMrjHiqZDTUASHg;spec/routing/0.1/create",
- "recipient-identifier" : "~4k1tvCU922atbe2gZaKY87" //Verkey Bob is going to use in his connection with Alice
````
I know I put this in but I think we should get rid of the inline comment here for the JSON, it just renders terribly on github

tplooker commented on text/agent-message-routing/README.md:
```diff
@@ -99,21 +101,21 @@ The DIDDoc assumed to be transmitted along side this connection request via micr
   "service": [
-    {"type": "Agency", "serviceEndpoint": "did:sov:agents-r-us" }
+    {"type": "Agency", "serviceEndpoint": "<C.did>" }
     // or "serviceEndpoint": "https://agents-r-us.com/" and add the #domain key (above)
```
And here

tplooker replied:
R.e remove comment

tplooker reviewed (changes requested):
Looks great!

Commit:
Move new connection sequence diagram

Move to the bottom of the "New Connection Example" section.

Also:

- Remove JSON comments as unsightly
- Clarify reference to UML diagram steps
